catalog/mvcc/cataloger_diff_test.go

```diff
@@ -11,6 +11,31 @@ import (
 	"github.com/treeverse/lakefs/testutil"
 )
 
+func TestCataloger_DiffEmpty(t *testing.T) {
+	ctx := context.Background()
+	c := testCataloger(t)
+	repository := testCatalogerRepo(t, ctx, c, "repo", "master")
+
+	// create N files and commit
+	commitChanges := func(n int, msg, branch string) {
```

Comment:
This structures the test more like `TestCataloger_Diff` below, so I'm keeping this style.  I think it is more readable to have the tests similar.  I do see that it makes this review harder because then you don't see the repeated `commitChanges` call.